stripe/src/main/java/com/stripe/android/model/StripeIntent.kt

```diff
@@ -158,6 +159,7 @@ interface StripeIntent : StripeModel {
      * Contains instructions for authenticating by redirecting your customer to another
      * page or application.
      */
+    @Deprecated("use {@link StripeIntent.NextActionData.RedirectToUrl}")
```

Comment:
Does this show up in Javadocs?

Reply:
Unfortunately, it doesn't seem to. Any thoughts on how we should handle this?

<img width="644" alt="Screen Shot 2020-05-20 at 12 01 03 PM" src="https://user-images.githubusercontent.com/47332718/82486504-d0120a00-9a91-11ea-8fdb-c9c3efbc56f1.png">

Reply:
That's fine, I was mainly wondering if it's appropriate to do `{@link }` in that string. In KDocs you can reference using `[StripeIntent.NextActionData.RedirectToUrl]`.

On that note, we should add a `@deprecated` string to the docstring as well.

Reply:
done